oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,218 @@
+
+- Our community is global.  Any time we could propose for this meeting would necessarily leave out a significant portion of our stakeholders and experts.
+
+- Our community is multi-faceted.  ADRs may have completely divergent sets of stakeholders.  How do we manage the guest list of such a meeting to ensure that the right voices are speaking on any given ADR, and that the right individuals are in the room?  It's a logistical nightmare given the size and scope of our community.
+
+- Our community is comprised of many independent organizations.  There are times when our component organizations may want to use the Advisory Process on "internal" decisions as well.  Do we create a separate forum meeting for those decisions?  How do we then ensure that we err on the side of having an open, public forum?  Again, logistically, this is quite difficult.
+
+Appendix A: Advisory Process Principles
+=======================================
+
+The creation of the Advisory Process is a fundamental shift in expectations of how we conduct our architectural process, and that shift requires an investment of time in order to be successful. The principles below describe the expectations, spirit and benefits of this process.
+
+Asynchronous First
+------------------
+
+Our community is global, and so we will prioritize asynchronous communication in the Architecture Advisory Process. This means it's important that individuals be able to participate in the process even if they cannot attend a synchronous meeting.  We believe there's value in getting together synchronously, but the process should prioritize getting the right input over having synchronous meetings.  That said, if the relevant stakeholders and experts are able to schedule an effective in person meeting, there's additional value in getting together synchronously when possible.  Note, however, that the decision makers *still have the responsibility of recording in their ADR the advice they received.*
+
+Decisions must document the advice they receive
+-----------------------------------------------
+
+One of the core tenets of the Advisory Process is that it is the decision maker's responsibility to record the advice they received, whether or not they chose to take it, and why.  We recognize that this is a time investment, but believe it will lead to better decisions, better documentation, and an approachable process where stakeholders and experts feel welcomed and encouraged to provide their invaluable insights.
+
+Decentralized decision making
+-----------------------------
+
+We assert that anyone can make an architectural decision, and that the only caveat is that they must consult those who are meaningfully affected by the decision, and those who have expertise in the area the decision is being taken.  The decision and all the advice they receive should be documented in an ADR, co-located with the relevant code following our established ADR guidelines in OEP-19.
```

Comment:
> they must consult those who are meaningfully affected by the decision, and those who have expertise in the area the decision is being taken

How do you know? How do you find these people? How are we encouraging 2U / OCM to recognize expertise may lay outside the building?

Reply:
How do you encourage people to recognize that they are actually an expert?

Reply:
This is part of what the decision maker's checklist is there to address.  I'm not any more concerned about 2U/OCM than I am about anyone else; the problem of finding expertise goes both ways and is, in general, a big gap for us all.  I also don't think this OEP needs to codify what we do to evangelize it within 2U.

Reply:
I don't have a good answer to how to encourage people to recognize that they're actually an expert.  I do think, though, that a bit of formalization (i.e., the checklist) will go a long way; there's nowhere to write it down, acknowledge it or encourage it right now.

Reply:
As I mention later on, I think maintainers should help with identifying experts.

I could see cross referencing OEP-55 in here - explaining here how maintainers are expected to help, and also adding that expectation back into OEP-55 (we're doing a similar thing with linking the Core Contributor & Maintainers OEPs)

Reply:
I think there's something here, yes.  @feanil and I were talking about whether, perhaps, the core of the "checklist" is actually just an index of maintainers with additional subject matter experts who aren't maintainers also called out.  Depending on the repo, pertinent SMEs may also include non-technical folks like UX, Product, Legal, a11y, etc.

Whether we need to codify what constitutes a subject matter expert is an interesting question.  🤔